frontend/src/components/Campaign/Campaign.config.ts

```diff
@@ -45,13 +40,26 @@ const TEST_PERSONAL_PROJECT:PersonalProjectEntry = {
     projectDate: "2019",
 }
 
+const currentDate = new Date();
```

Comment:
don't need to make this a config constant u can just directly use new Date() in the acutal code, constants are more for template mocks or for strings